ckb-vm-bench-scripts/Makefile

```diff
@@ -63,8 +63,8 @@ build:
 
 report:
 	@for contract in $(wildcard contracts/*); do \
-		du -h build/$(MODE)/$$(basename $$contract); \
-		ckb-debugger --bin build/$(MODE)/$$(basename $$contract); \
+		du -h build/$(MODE)/$$(basename $$contract)_ckbvm; \
```

Comment:
I recommend also a `du` line without `-h` option, some cases might be more precise expressing in raw bytes.

Reply:
`du` displays in kbytes by default. Let me add a `du -b` line.

> Display values are in units of the first available SIZE from --block-size,
and the DU_BLOCK_SIZE, BLOCK_SIZE and BLOCKSIZE environment variables.
Otherwise, units default to 1024 bytes (or 512 if POSIXLY_CORRECT is set).